src/main/java/frc/robot/commands/ScoreStageTwoColorSwitch.java

```diff
@@ -37,7 +37,7 @@ public ScoreStageTwoColorSwitch(ColorSensor c, ControlPanelSubsystem p) {
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    controlPanel.wheelUp();
+    controlPanel.setMotorSpeed(10);
```

Comment:
I don't _think_ this is strictly required in `initialize()`.  I think the first time a command is run it will execute:

- `initialize()`
- `execute()`
- `isFinished()`
- `end()` if `isFinished()`

```suggestion
```

I don't think it hurts to have it here either.